config/routes.rb

```diff
@@ -6,4 +6,6 @@
 
   # Uncomment this line to activate the endpoint for the Square hosted checkout workflow.
   # post 'square_checkout', to: '/solidus_square/callback_actions#square_checkout'
+
+  patch "square_update", to: '/solidus_square/webhooks#update'
```

Comment:
IMO it should be disabled by default and we have to add some documentation to the README file to explain how to enable it.
Instead of `square_update` what do you think to call it `webhooks/square` ?

Reply:
https://github.com/nebulab/solidus_square/pull/36/commits/a4c128ae22ac7488ed05932e0b72847d6502933e

https://github.com/nebulab/solidus_square/pull/36/commits/52ec575ef242ad0c119cdc1786a28732e22d87c3